docs/types/reference-types.rst

```diff
@@ -85,8 +85,10 @@ Data locations are not only relevant for persistency of data, but also for the s
             // The following does not work; it would need to create a new temporary /
             // unnamed array in storage, but storage is "statically" allocated:
             // y = memoryArray;
-            // This does not work either, since it would "reset" the pointer, but there
-            // is no sensible location it could point to.
+            // On the other hand: "delete y" is not valid, as assignments to local variables
```

Comment:
"On the other hand" seems to suggest that the situation is different here.

Reply:
Woops, yes, those words should be omitted.